src/pages/developing/angular-tutorial/step-1.mdx

```diff
@@ -797,22 +300,32 @@ package.
 
 </InlineNotification>
 
-#### Import icons
+#### Import and register icons
 
-Now let's import the icons from our `@carbon/icons-angular` package. In
-`app.module.ts`, `app.component.spec.ts` and `header.component.spec.ts`, we need
-to import each individual icon we will use.
+Now let's import the icons from our `@carbon/icons` package. In `app.module.ts`,
+`app.component.spec.ts` and `header.component.spec.ts`, we need to import each
+individual icon we will use and register them with the `IconService`
 
 <!-- prettier-ignore-start -->
 ```javascript path=src/app/app.module.ts,src/app/app.component.spec.ts,src/app/header/header.component.spec.ts
-import { NotificationModule } from '@carbon/icons-angular';
-import { UserAvatarModule } from '@carbon/icons-angular';
-import { AppSwitcherModule } from '@carbon/icons-angular';
+import Notification16 from '@carbon/icons/es/notification/16';
+import UserAvatar16 from '@carbon/icons/es/user--avatar/16';
+import AppSwitcher16 from '@carbon/icons/es/app-switcher/16';
 ```
 <!-- prettier-ignore-end -->
 
+In the `AppModule` class we'll add a constructor that provides us with an
+instance of `IconService` and call `registerAll` with an array of the icons we
+need to use.
+
 ```javascript path=src/app/app.module.ts,src/app/app.component.spec.ts,src/app/header/header.component.spec.ts
-imports: [NotificationModule, UserAvatarModule, AppSwitcherModule];
+constructor(protected iconService: IconService) {
+	iconService.registerAll([
+    Notification16,
```

Comment:
Mixed tabs and spaces :)

Reply:
🤦 oops ... Can you tell which one I copied out of the components-angular source?